generators/client/templates/react/package.json

```diff
@@ -12,7 +12,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-hook-form": "7.28.1",
-    "react-jhipster": "0.19.0",
+    "react-jhipster": "jhipster/react-jhipster",
```

Comment:
Does this need to be updated before merging?

Reply:
IMO we can merge as it is now.
The install time increases a bit, but works as expected.
CC @deepu105